test/source/smime-parse.ts

```diff
@@ -0,0 +1,42 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+import * as ava from 'ava';
+import { SmimeKey } from '../../extension/js/common/core/crypto/smime/smime-key.js';
```

Comment:
You can move this into `unit.ts` file and then import from a local core folder (symlink copy) in the test folder.

The "core" folder is the same, but the adjecent "platform" folder will provide platform specific stuff for the core to work on node.